ibm/acctest/acctest.go

```diff
@@ -1336,6 +1336,12 @@ func init() {
 		Pi_instance_name = "terraform-test-power"
 		fmt.Println("[INFO] Set the environment variable PI_INSTANCE_NAME for testing ibm_pi_instance resource else it is set to default value 'terraform-test-power'")
 	}
+	Pi_instance_id = os.Getenv("PI_INSTANCE_ID")
+	if Pi_instance_id == "" {
+		Pi_instance_id = "terraform-test-power"
+		fmt.Println("[INFO] Set the environment variable PI_PVM_INSTANCE_ID for testing Pi_pvm_instance_id  else it is set to default value 'terraform-test-power'")
```

Comment:
```suggestion
		fmt.Println("[INFO] Set the environment variable PI_INSTANCE_ID for testing Pi_instance_id  else it is set to default value 'terraform-test-power'")
```
I'm not sure if that's fixed by pulling the latest code

Reply:
updated